synapse/app/media_repository.py

```diff
@@ -151,6 +150,15 @@ def start(config_options):
 
     assert config.worker_app == "synapse.app.media_repository"
 
+    if config.enable_media_repo:
+        sys.stderr.write(
+            "\nenable_media_repo must be disabled in the main synapse process"
+            "\nbefore the media repo can be run in a separate worker."
+            "\nPlease add ``enable_media_repo: false`` to the main config"
+            "\n"
+        )
+        sys.exit(1)
```

Comment:
`quit_with_error` in `_base.py` is a thing

Reply:
done f24dc52